radio/src/translations/pl.h.txt

```diff
@@ -79,8 +81,8 @@
 #define LEN_TRNCHN             "\003"
 #define TR_TRNCHN              "KN1KN2KN3KN4"
 
-#define LEN_UART3MODES         "\017"
-#define TR_UART3MODES          "Wyłącz\0        ""S-Port Mirror\0 ""Telemetry\0     ""SBUS Trainer\0  ""Debug\0"
+#define LEN_UART3MODES         "\015"
+#define TR_UART3MODES          "Wyłącz\0        ""S-Port Kopia ""Telemetria\0  ""Trener SBUS\0 ""Debug\0"
```

Comment:
This is not good. The defined length of each string is \015 (octal) which is 13 (decimal). The first string `"Wyłącz\0        "` is 15 characters.